assets/csv/entities.csv

```diff
@@ -4,7 +4,7 @@ id,name,href,profile_src,description
 3,TMGOX,https://www.tmgox.com/,assets/images/logos/tmgox-logo.jpg,
 4,GPU.one,https://gpu.one/,assets/images/logos/gpuOne-white.png,
 5,Mega Pool,https://www.megapool.info,assets/images/logos/Mega-Pool-Logo-trans.png,
-6,BlockCypher,https://www.blockcypher.com,assets/images/logos/blockcypher_logo_white.svg,
+7,Grinmint by BlockCypher,https://www.grinmint.com/img/grinmint_logo_square.svg,
```

Comment:
Should be a `6` here. Other than that, LGTM! 😊

Reply:
Indeed not sure what happened let me fix this.